### PR TITLE
UI searchFilm by fenres

### PR DESCRIPTION
**Pull request by vitconpc**
Code UI search by Genres Screen

### CODE REVIEW

**Comment by vitconpc** [comment]
ready

File: app/src/main/java/vn/com/framgia/movie_db26/screen/list/FindGenresModel.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+                    public void accept(Throwable throwable) throws Exception {
+                        Toast.makeText(mContext, R.string.error_network, Toast.LENGTH_SHORT).show();
+                    }
+                });
+        mCompositeDisposable.add(disposable);
+    }
+
+    @Override
+    public void onClickItem(Film film) {
+        Intent intent = new Intent(mContext, DetailActivity.class);
```
Comment (tuannguyen38) [inline app/src/main/java/vn/com/framgia/movie_db26/screen/list/FindGenresModel.java]: convention 2.11

File: app/src/main/res/layout/activity_find_genres.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                app:srcCompat="@drawable/icon_app"
+                android:layout_width="@dimen/dp_48"
+                android:layout_height="match_parent"
+                android:layout_marginBottom="@dimen/dp_8"
+                android:layout_marginLeft="@dimen/dp_8"
+                android:layout_marginStart="@dimen/dp_8"
+                android:layout_marginTop="@dimen/dp_8" />
+
+            <TextView
+                android:id="@+id/textView"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_find_genres.xml]: convention 3.2

File: app/src/main/res/layout/item_film_search.xml [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                app:layout_constraintStart_toStartOf="@+id/image_search_item_card"
+                app:layout_constraintTop_toBottomOf="@+id/text_release_date"
+                app:layout_constraintVertical_bias="0.0" />
+
+            <ImageView
+                android:id="@+id/image_float_favorite"
+                android:layout_width="@dimen/dp_40"
+                android:layout_height="@dimen/dp_40"
+                android:layout_marginEnd="24dp"
+                android:layout_marginRight="24dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_film_search.xml]: 148 149 hardcode

File: app/src/main/res/layout/activity_find_genres.xml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            recyclerAdapterVertical="@{viewModel.findGenresAdapter}"
+            android:id="@+id/recycler_item_search"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            android:layout_marginBottom="@dimen/dp_8"
+            android:layout_marginEnd="@dimen/dp_8"
+            android:layout_marginLeft="@dimen/dp_8"
+            android:layout_marginRight="@dimen/dp_8"
+            android:layout_marginStart="@dimen/dp_8"
+            android:layout_marginTop="8dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_find_genres.xml]: hardcode

File: app/src/main/java/vn/com/framgia/movie_db26/screen/list/FindGenresModel.java [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        mContext.startActivity(getIntent(mContext, film));
+    }
+
+    public static Intent getIntent(Context context, Film film) {
+        Intent intent = new Intent(context, DetailActivity.class);
+        intent.putExtra(Constants.ID_FILM, film.getId());
+        return intent;
+    }
+
+
```
Comment (tuannguyen38) [inline app/src/main/java/vn/com/framgia/movie_db26/screen/list/FindGenresModel.java]: remove bl

**Pull request merged**